### PR TITLE
fix(meta): semantic-release on actions

### LINKED ISSUE #5514: Meta: move semantic-release from circle to actions
Let's move releasing to Actions and then deprecate CircleCI use.

- [x] Define npm release token in Actions secrets (the embedded GitHub token in Actions should be enough for the GitHub part)
- [ ] Add semantic releasing to Node 12 on Linux in Actions
- [ ] Disable releasing on CircleCI

Note: I don't want to use any third party action for this, because of the risk of it stealing credentials.

### PR DESCRIPTION
**Pull request by rarkins**
Closes #5514

### CODE REVIEW

File: .github/workflows/test-and-release.yml [hunk 1]
```diff
@@ -126,3 +126,56 @@ jobs:
 
       - name: E2E Test
         run: yarn test-e2e
+
+  release:
+    name: release
+    needs: test
+    runs-on: ubuntu-latest
+    # if: github.ref == 'refs/heads/master'
```
Comment (rarkins) [inline .github/workflows/test-and-release.yml]: Commented out for now in order to try to test this in the PR itself

**Review by viceice — APPROVED** [review]

**Comment by ViceIce** [comment]
should we use the `renovate-release` token for release? so now we have the `github-actions` bot as releaser

**Comment by rarkins** [comment]
I don't see much downside in having the github-actions bot do it though, do you see any? One less token to manage

**Comment by ViceIce** [comment]
No, it's fine. just to mention. :-)

**Pull request merged**